src/components/organisms/Content/Routing.tsx

```diff
@@ -2,38 +2,21 @@ import React from 'react'
 import { Switch, Route, Redirect } from 'react-router-dom'
 
 import About from 'components/pages/About'
-import Nemeses from 'components/pages/Nemeses'
-import Mages from 'components/pages/Mages'
-import Supply from 'components/pages/Supply'
 import Settings from 'components/pages/Settings'
 import ContentCustomization from 'components/pages/Settings/Expansions/ContentCustomization'
 import TurnOrder from 'components/pages/TurnOrder'
 import Expeditions from 'components/pages/Expeditions'
 import Expedition from 'components/pages/Expeditions/Expedition'
-import BasicNemesisCards from 'components/pages/BasicNemesisCards'
+import Randomizer from 'components/pages/Randomizer'
 
 type Props = {}
 
 const Routing = (props: Props) => {
   return (
     <Switch>
-      <Redirect exact={true} from="/" to="/randomizer/nemesis" />
+      <Redirect exact={true} from="/" to="/randomizer/" />
```

Comment:
Is '/' at the end of the route /randomizer neccessary?

Reply:
it's not necessary, but does not hurt either (I think)

Reply:
okay, because it's the only route where we put this on at the end